src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,111 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private static final int MIN_AGE = 18;
+    private static final int MIN_PASSWORD = 6;
+    private static final int STORAGE_LIST_SIZE = 2;
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+    private User user;
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setId(321123L);
+        user.setPassword("hawkIsGood");
+        user.setLogin("SomeLogin");
+        user.setAge(30);
+    }
+
+    @Test
+    void register_Young_IsNotOk() {
+        user.setAge(16);
+        Assertions.assertThrows(RuntimeException.class, () -> registrationService.register(user),
+                "Expected exception when user age is less than required");
+    }
+
+    @Test
+    void register_Old_IsOk() {
+        User register = registrationService.register(user);
+        assertTrue(register.getAge() > MIN_AGE, "Good! You age perfect for registration");
+    }
+
+    @Test
+    void registerAge_Null_IsNotOk() {
+        user.setAge(null);
+        Assertions.assertThrows(RuntimeException.class, () -> registrationService.register(user),
+                "Age is null. Give some age for user");
+    }
+
+    @Test
+    void registerPassword_SizeIsOk() {
+        User register = registrationService.register(user);
+        assertTrue(register.getPassword().length() > MIN_PASSWORD, "Good! Password have nice size");
+    }
+
+    @Test
+    void registerPassword_Size_IsNotOk() {
+        user.setPassword("abc");
+        Assertions.assertThrows(RuntimeException.class, () -> registrationService.register(user),
+                "Password to small. Try agan");
+    }
+
+    @Test
+    void registerPassword_Null_IsNotOk() {
+        user.setPassword(null);
+        Assertions.assertThrows(RuntimeException.class, () -> registrationService.register(user),
+                "Password is null. Give some password for user");

```

Comment:
<img width="801" alt="Screen Shot 2022-11-21 at 5 29 57 PM" src="https://user-images.githubusercontent.com/90692308/203094086-78291ea6-c323-43ed-b9dd-d6e35137eafd.png">
Fix it everywhere